frontend/public/components/edit-yaml.jsx

```diff
@@ -64,19 +66,40 @@ const WithYamlTemplates = (Component) =>
     );
   };
 
+const getModel = (models, obj) => {
+  if (_.isEmpty(obj) || !models) {
+    return null;
+  }
+  return models.get(referenceFor(obj)) || models.get(obj.kind);
+};
+
+export const createResources = (objs, models, isDryRun) => {
```

Comment:
Same comment here. If this were a method on the class, you wouldn't need to pass `models`.

Reply:
Yeah, I changed this such that they are both class methods where the createResources is passed to the new results component as a props, and I don't have to shuffle models around anymore.